UPGRADING.rst

```diff
@@ -11,6 +11,42 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#2246 Add deployment incarnation counter
+========================================
+
+See upgrading instructions for #2143 below.
+
+
+#2143 Merge service accounts for indexer and service
+====================================================
+
+1.  Before upgrading to this commit, run ::
+
+        make -C terraform validate
+        (cd terraform && terraform destroy -target=google_service_account.indexer)
+
+2.  Upgrade to this commit or a later one.
+
+3.  Let someone who can administer the SAM group that controls access to TDR
+    know of the renamed service account via Slack. The administrator will need
+    to replace the old service account email with the new one. For example,
+
+    replace ::
+
+        azul-ucsc-indexer-foo@human-cell-atlas-travis-test.iam.gserviceaccount.com
+
+    with ::
+
+        azul-ucsc-0-foo@human-cell-atlas-travis-test.iam.gserviceaccount.com
```

Comment:
This can't work. At this point this account doesn't exist yet so the SAM UI will refuse to add it to the group.

Reply:
I've been able to add accounts to JadeStewards-dev that are not registered with SAM. The UI does not block me in JadeStewards-dev, although it does pop up this notification. I've not run into issues with authorization/authentication when adding the account into the JadeStewards-dev group before performing a deployment that changes service accounts.

Although if someone changes this at the Broad our documentation could be misleading or worse broken. I think moving this as a second to last step to complete would be ok, then informing the user to confirm access with `make -C terraform sam`

![Screen Shot 2020-10-22 at 9 26 17 AM](https://user-images.githubusercontent.com/1947534/96901903-b2820b80-1448-11eb-8aa0-676c23c07b81.png)

Reply:
092b742 resolves this.